src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+:root {
+  box-sizing: border-box;
+  --main-text-color: #3d4e61;
+  --border-color: #e1e7ed;
+  --background-color: #fff;
+}
+
+.search-form {
+  display: block;
+  position: relative;
+  margin-top: 20px;
+}
+
+.search-form__input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+
+  font-family: "Avenir", sans-serif;
```

Comment:
Move font-family into `html` selector

Reply:
If I move it there, I am not able to pass the test. As it seems to me, that the font family does not get inherited to the input